src/common/Types.ts

```diff
@@ -368,6 +368,7 @@ export interface CommonAccessibilityProps {
 
     // Desktop only.
     tabIndex?: number;
+    valueNow?: number;
```

Comment:
New props should be documented. In this case, please add it to the 'accessibility.md' documentation. (I alphabetized the props in the docs, BTW.) There's no need to modify the documentation on the 0.51.x branch because only the master branch docs are published.

Reply:
Updated doc with valueNow prop info.